anymail/backends/scaleway.py

```diff
@@ -0,0 +1,163 @@
+            api_url += "/"
+        super().__init__(api_url, **kwargs)
+
+    def build_message_payload(self, message, defaults):
+        return ScalewayPayload(message, defaults, self)
+
+    def parse_recipient_status(self, response, payload, message):
+        parsed_response = self.deserialize_json_response(response, payload, message)
+        statuses = {}
+        try:
+            emails = parsed_response["emails"]
+        except (KeyError, TypeError):
+            raise AnymailAPIError(
+                "Invalid response from Scaleway API",
+                email_message=message,
+                payload=payload,
+                response=response,
+                backend=self,
+            )
+
+        for email_info in emails:
+            recipient = email_info.get("mail_rcpt")
+            message_id = email_info.get("id")
+            status = email_info.get("status")
+            status_map = {
+                "new": "queued",
+                "sending": "queued",
+                "sent": "sent",
+                "failed": "failed",
+                "canceled": "failed",
+            }
+            anymail_status = AnymailRecipientStatus(
+                message_id=message_id, status=status_map.get(status, "unknown")
+            )
+            if recipient:
+                statuses[recipient] = anymail_status
+        return statuses
+
+
+class ScalewayPayload(RequestsPayload):
+    def __init__(self, message, defaults, backend, *args, **kwargs):
+        self.project_id = backend.project_id
+        http_headers = kwargs.pop("headers", {})
+        http_headers["X-Auth-Token"] = backend.api_key
+        http_headers["Content-Type"] = "application/json"
+        super().__init__(
+            message, defaults, backend, headers=http_headers, *args, **kwargs
+        )
+
+    def get_api_endpoint(self):
+        return "emails"
+
+    def init_payload(self):
+        self.data = {
+            "to": [],
+            "cc": [],
+            "bcc": [],
+            "additional_headers": [],
+            "attachments": [],
+        }
+        if self.project_id:
+            self.data["project_id"] = self.project_id
+
+    def set_from_email(self, email):
+        self.data["from"] = {
+            "email": email.addr_spec,
+            "name": email.display_name or None,
+        }
+
+    def set_recipients(self, recipient_type, emails):
+        if emails:
+            self.data[recipient_type] = [
+                {"email": email.addr_spec, "name": email.display_name or None}
+                for email in emails
+            ]
+
+    def set_subject(self, subject):
+        if subject:
+            self.data["subject"] = subject
+
+    def set_text_body(self, body):
+        if body:
+            self.data["text"] = body
+
+    def set_html_body(self, html):
+        if html:
+            self.data["html"] = html
+
+    def add_attachment(self, attachment):
+        self.data.setdefault("attachments", []).append(
```

Comment:
It looks like Scaleway's API doesn't have any way to specify inline images, so that counts as an "unsupported feature":

```suggestion
        if attachment.inline:
            self.unsupported_feature("inline attachments")
        self.data.setdefault("attachments", []).append(
```

(If the user is ignoring unsupported feature errors, this will fall through and handle the inline attachment as a regular attachment.)